internal/data/wallets_test.go

```diff
@@ -137,6 +137,53 @@ func Test_WalletModelGetAll(t *testing.T) {
 	})
 }
 
+func Test_WalletModelFindWallets(t *testing.T) {
+	dbt := dbtest.Open(t)
```

Comment:
just a question, give these are unit tests shouldnt we be mocking the database?

Reply:
If we were testing a method that performs some sort of business logic I'd agree that we can mock the database. 

But we are testing the Model (WalletModel in this case) which only performs DB operations. It makes sense to use a test db to run these ones as there isn't much else to test. That's the pattern we've been following for testing all Model operations.